central/graphql/resolvers/loaders/namespaces.go

```diff
@@ -0,0 +1,147 @@
+package loaders
+
+import (
+	"context"
+	"reflect"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/central/namespace/datastore"
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/env"
+	"github.com/stackrox/rox/pkg/search"
+	"github.com/stackrox/rox/pkg/sync"
+)
+
+var namespaceLoaderType = reflect.TypeOf(storage.NamespaceMetadata{})
+
+func init() {
+	if env.PostgresDatastoreEnabled.BooleanSetting() {
```

Comment:
Why is the loader behind postgres feature flag. The namespace datastore has always existed. You do not need to guard the loader registration behind the feature flag.